test/python/tensorflow/tests_linux_ie_cpu.txt

```diff
@@ -0,0 +1,401 @@
+# Manifest to specify included & excluded tests
+# Comment lines start with #
+# This manifest is consumed by .../test/python/tensorflow/tf_unittest_runner.py
+
+#  Few examples of valid regex/pattern :
+#    *test* --> module
+#    *test*.*test* --> module.class
+#    *_op_test*.*[Tt]est* --> e.g. pad_op_test.PadOpTest.testInputDims
+#    math_ops_test.DivNoNanTest.testBasic --> module.class.testfunc
+#    math_ops_test.DivNoNanTest.* --> module.class.testfunc
+#    math_ops_test.D* --> module.class
+#    math_ops_test.* --> module.class
+#    math_*_test --> module
+#    math_*_*_test --> module
+#    math*_test --> module
+
+###################################################
+# First specify the included tests/patterns/regex
+# or, just include a filename relative to this file's path
+
+include python_tests_list.txt # --> include a filename relative to this file's path
```

Comment:
python_tests_list.txt has some tests commented, so those won't be run for IE anyway - correct?